src/main/migrations/20240830143811_Hydra_2_0_3.ts

```diff
@@ -98,6 +98,7 @@ export const Hydra2_0_3: HydraMigration = {
           table.text("downloadsPath");
           table.text("language").notNullable().defaultTo("en");
           table.text("realDebridApiToken");
+          table.text("torboxApiToken");
```

Comment:
you actually need to create a new migration file, and import it in `src/main/knex-client.ts`